Neos.Neos/Classes/Domain/Service/NodeDuplicationService.php

```diff
@@ -179,6 +201,34 @@ public function calculateCopyNodesRecursively(
         return $commands;
     }
 
+    private function resolveUriPathSegmentForCopy(
```

Comment:
thats nice and intelligent;) i think we should also improve the uri path segment for new nodes like that if no path is set? https://github.com/neos/neos-ui/blob/70cadd43e3f17b0c47cd39d251165a739eda910f/Classes/Infrastructure/Neos/UriPathSegmentNodeCreationHandlerFactory.php#L30